website/content/docs/auth/login-mfa.mdx

```diff
@@ -0,0 +1,184 @@
+---
+layout: docs
+page_title: Multi-Factor Authentication (MFA) for Login - Auth Methods
+description: |-
+  Multi-factor authentication (MFA) is supported for several authentication
+  methods.
+---
+# Login MFA
+
+Vault has support for Multi-factor Authentication (MFA) for authenticating to
+an auth method, using different authentication types. We use the term `Login MFA` to distinguish
+this feature and the [Vault Enterprise MFA](/docs/enterprise/mfa).
+Login MFA is built on top of the Identity system of Vault.
+
+## MFA Types
+
+Login MFA in Vault can be of the following types.
+
+- `Time-based One-time Password (TOTP)` - If configured and enabled on a login path,
+  this would require a TOTP passcode along with a Vault token, to be presented
+  while invoking the API login request. The passcode will be validated against the
+  TOTP key present in the identity of the caller in Vault.
+
+- `Okta` - If Okta push is configured and enabled on a login path, then the enrolled
+  device of the user will get a push notification to approve or deny the access
+  to the API. The Okta username will be derived from the caller identity's
+  alias.
+
+- `Duo` - If Duo push is configured and enabled on a login path, then the enrolled
+  device of the user will get a push notification to approve or deny the access
+  to the API. The Duo username will be derived from the caller identity's
+  alias. Note that Duo could also be configured to use passcodes for authentication.
+
+- `PingID` - If PingID push is configured and enabled on a login path, then the
+  enrolled device of the user will get a push notification to approve or deny
+  the access to the API. The PingID username will be derived from the caller
+  identity's alias.
+
+## Login MFA Procedure
+
+Login MFA can be configured to further secure authenticating to an auth method. To enable login
+MFA, an MFA method needs to be configured. Please see [Login MFA API](/api/secret/identity/mfa) for details
+on how to configure an MFA method. Once an MFA method is configured, using the returned unique MFA method ID,
+an operator can configure an MFA enforcement. Please see [Login MFA Enforcement API](/api/secret/identity/mfa/login-enforcement)
+for details on how to configure an MFA enforcement config. MFA could be enforced for an entity, a group of
+entities, a specific auth method accessor, or an auth method type. A login request that matches
+any of the MFA enforcement restrictions is subject to further MFA validation,
+like a one-time passcode, before being authenticated.
+
+Note that both MFA methods like (Okta, Duo, and PingID) and MFA enforcement configuration
+could use mount accessors. However, there is no particular correlation between the two accessors. In other words, the
```

Comment:
Should we clarify the "no particular correlation" part here a bit? Since there does need to be some relation to map the user trying to login to the accessor on the method, iiuc. Maybe something like
```The accessors do not necessarily need to be the same for the 2 cases. However, the user trying to login on an MFA enforced mount would need to have an alias on the accessor in the MFA method itself, in the case that entity/alias templating is required to render out a username``` ?